### PR TITLE
[proto-render-setup Part 3]: Base Testing logic, ideal TODOs

### PR DESCRIPTION
**Pull request by Marcial1234**
Part of #247, all tests pass depending on #254 definitions. Ready for nits!

End of setup batch. Will dump element-by-element rendering + template + proto changes for the next stacked PRs batch.

### CODE REVIEW

File: third_party/render_html.go [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 )
 
 func init() {
-	funcMap := template.FuncMap{
-		"safeHTML": func(s string) htmlTemplate.HTML {
-			return template.HTML(s)
-		},
-	}
-	html = template.Must(template.New("master").Funcs(funcMap).ParseGlob(htmlTmplsDir))
+	html = template.Must(template.New("master").ParseGlob(htmlTmplsDir))
```
Comment (shawnbuso) [inline third_party/render_html.go]: Is this supposed to be undoing stuff that was in PR 2? If so should we just remove it from PR 2?

File: third_party/render_md_test.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package devrel_tutorial
+
+import (
+	"testing"
+	"github.com/googlecodelabs/tools/claat/util"
+)
+
+func TestRenderHeadingMd(t *testing.T) {
+	h2 := &Heading{
+    Level: 2,
```
Comment (shawnbuso) [inline third_party/render_md_test.go]: Fix indentation here and below

Comment (Marcial1234) [reply third_party/render_md_test.go]: aha! good catch! I'm using Sublime text with same size tabs as two-spaces. Didn't catch this. This repo uses tabs for default spacing, for some reason.

File: third_party/render_html.go (hunk 1 above)
Comment (Marcial1234) [reply third_party/render_html.go]: I found a better way to do this, so it's not an undo. But the change definitely needs to be on Part 2. Will clean that one in general a bit.

**Comment by Marcial1234** [comment]
Hold on further reviewing until Design Doc is approved, closing for now

**Pull request closed without merging**